README.md

```diff
@@ -1,2 +1,4 @@
 ### Build Status
 [![Build Status](https://dev.azure.com/xamarin/public/_apis/build/status/xamarin/Xamarin.PropertyEditing/Xamarin.PropertyEditing%20(Public)?branchName=main)](https://dev.azure.com/xamarin/public/_build/latest?definitionId=51&branchName=main)
```

Comment:
Would it be reasonable to remove this "Build Status" indicator? I assume the build pipeline would be disabled or removed.

Reply:
We're still going to use this for older UITools branches, so we'll need to keep the pipelines and I think there's a little bit of value in keeping the build status indicator here.